src/modules/CurrencyExchange/CurrencyExchange.tsx

```diff
@@ -0,0 +1,22 @@
+import { FC, useState } from "react";
+import { ReactNode } from "react";
+import { Wrapper } from "../../components/Wrapper";
+import { currencyData } from "../../data";
+import { CurrencyDisplay } from "../CurrencyDisplay";
+import { CurrencyList } from "../CurrencyList";
+import { CurrencyType } from "../../types";
+
+type WrapperProps = {
+  children?: ReactNode | ReactNode[];

```

Comment:
Jeśli nie potrzebujesz `children` to nie trzeba go dodawać do każdych prop-sów

Reply:
jak wyżej children prop usunięte gdzie nie trzeba.